group_vars/all.yml

```diff
@@ -12,6 +12,7 @@ ansible_ssh_key: "{{ ansible_user_dir }}/.ssh/id_rsa"
 osp_release: 13
 osp_puddle: passed_phase2
 controller_count: 3
+dns_ip: 10.1.32.3
```

Comment:
Not needed

Reply:
Also you can add badfish_image as a variable and use it when you pull the podman image.